src/vue/pages/sale-details/SaleStateWidget.vue

```diff
@@ -39,13 +39,23 @@
       })"
     />
 
-    <vue-markdown
-      class="sale-state-widget__days-to-go"
-      :source="'sale-details.days-to-go' | globalize({
-        days: sale.daysToGo
-      })"
-    />
-
+    <template v-if="sale.daysToGo < 0">
```

Comment:
change the condition. not `daysToGo` should be checked, but `daysAfterEnd`. because negative amount of `daysToGo` looks a bit strange and its much more sexy to hide it inside getter. the template should act something like that: "if daysAfterEnd is positive, I assume that I should show days-after-end label, otherwise days-to-go label should be shown. I cannot be sure is it bug or whatever caused daysToGo being negative"